src/styles/blocks/card.scss

```diff
@@ -0,0 +1,92 @@
+.card {
+  width: 200px;
+  box-sizing: border-box;
+  padding: 32px 16px 16px;
+  border: 1px solid $color-light-gray;
+  border-radius: 5px;
+  transition: all 0.3s;
+
+  &__image {
+    display: flex;
+    height: 134px;
+    width: 160px;
+    margin: 0 auto;
+  }
+
+  &__title {
+    margin-top: 40px;
+    font-size: 12px;
+    font-weight: 500;
+    line-height: 18px;
+    color: $text-color-navy-blue;
+  }
+
+  &__product-code {
+    margin-top: 4px;
+    font-size: 10px;
+    line-height: 14px;
+    color: $color-dim-gray;
+  }
+
+  &__rating {
+    display: flex;
+    justify-content: space-between;
+    align-items: center;
+    margin-top: 16px;
+
+    &-review {
+      font-size: 10px;
+      color: $text-color-navy-blue;
+    }
+  }
+
+  &__price {
+    display: flex;
+    justify-content: space-between;
+    margin-top: 24px;
+
+    &-text {
+      line-height: 18px;
+      color: $color-dim-gray;
+    }
+
+    &-value {
+      font-size: 16px;
+      font-weight: 700;
+      line-height: 18px;
+      color: $text-color-navy-blue;
+    }
+  }
+
+  &__button {
+    display: flex;
+    flex-direction: column;
+    align-items: center;
+    justify-content: center;
+    margin-top: 16px;
+    height: 40px;
+    font-size: 14px;
+    font-weight: 700;
+    line-height: 16px;
+    text-transform: uppercase;
+    text-decoration: none;
+    color: $color-white;
+    border-radius: 5px;
+    background-color: $color-vivid-cerulean;
+    transition: all 0.3s;
+
+    &:hover {
+      border: 1px solid $color-vivid-cerulean;
+      background-color: $color-white;
+      color: $color-vivid-cerulean;
+    }
+  }
+
+  &:hover {
+    transform: scale(1.2);
+  }
+
+  &:hover &__title {
+    color: $color-steel-blue;
+  }
```

Comment:
The hover effect on the card title changes its color to $color-steel-blue. Ensure this color provides sufficient contrast against the background for readability.